src/components/protocol/src/service_type.cc

```diff
@@ -63,6 +63,7 @@ ServiceType ServiceTypeFromByte(uint8_t byte) {
   return valid_type ? type : kInvalidServiceType;
 }
 
+// cppcheck-suppress unusedFunction
 uint8_t ServiceTypeToByte(ServiceType type) {
```

Comment:
@Ypostolov It seems like the function is unused. Please, check it and remove if it is true.